develop/sdk/examples.md

```diff
@@ -65,7 +66,6 @@ func main() {
 	resp, err := cli.ContainerCreate(ctx, &container.Config{
 		Image: "alpine",
 		Cmd:   []string{"echo", "hello world"},
-		Tty:   true,
```

Comment:
Was there a reason for removing this?

Reply:
not that I recall! (...this was a while ago)